.github/workflows/wheels.yml

```diff
@@ -1,12 +1,14 @@
 name: Build
 
-on:
```

Comment:
I changed this because otherwise every push triggered the workflow, and updating a PR triggered the workflow twice

Reply:
Reverted this part